tutorial/index.jade

```diff
@@ -1,10 +1,155 @@
+
+  .step(data-title='Send a Verification Token', data-file='models/User.js', data-highlight='73-101')
+    :markdown
+      ## Send a Verification Token
+
+      When it comes time to actually send the user a verification code, we do that in our model function here.
+
+      Before sending the code, an [Authy user](https://docs.authy.com/#section-Enabling_two-factor_on_a_user) needs to exist as a pair for our `User` model in the database.  If the `authyId` for our user instance hasn't been set, we use the Authy API client to create an associated Authy user and store that ID in our database.
+
+      Once the user has an `authyId`, we can actually send a verification code to that user's mobile phone.
+
+      ---
+
+      **See Also:**
+      * [Node Client for Authy API](//github.com/authy/node-authy)
+
+  .step(data-title='Verifying the Code: Controller', data-file='controllers/users.js', data-highlight='95-147')
+    :markdown
+      ## Verifying the Code: Controller
+
+      This controller function handles the form submission. It's a little longer than the others, but it has a lot to do. It needs to:
+
+      * Load a `User` model for the current verification request
+      * Use an instance function on the model object to verify the code that was entered by the user
+      * If the code entered was valid, flip a boolean flag on the user model to indicate the account was verified
+
+      Let's look at the `User` model to check out the instance method that handles verifying the code with Authy next.
+
+  .step(data-title='Verifying the Code: Model', data-file='models/User.js', data-highlight='103-109')
+    :markdown
+      ## Verifying the Code: Model
+
+      This instance function is a thin wrapper around the Authy client function that sends in a candidate password to be verified. 
+      We call Authy, and then immediately call a passed callback function with the result.
+
+  .step(data-title='Re-sending The Code', data-file='controllers/users.js', data-highlight='65-93')
+    :markdown
+      ## Re-sending the Code
+
+      This controller function loads the `User` model associated with the request and then uses the same instance function we defined earlier to resend the code. Pretty straightforward!
+
+      To wrap things up, let's implement the last step where we confirm that the user's account has
+      been verified with a text message and a success page.
+
+  .step(data-title="Show User Details Page", data-file="controllers/users.js", data-highlight="149-167")
+    :markdown
+      ## Show User Details Page
+
+      This controller function renders a Jade template that contains the user's full name, and indicates whether or not they are verified by checking the user's `verified` property.
+
+      This should suffice for confirmation in the browser that the user has been verified. Let's see how we might send that text message next.
```

Comment:
Adding a reference to [`Model#findById`](http://mongoosejs.com/docs/api.html#model_Model.findById) would be good here.